bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/internal/handler/GenericMQTTThingHandler.java

```diff
@@ -100,14 +100,15 @@ public void dispose() {
         // Remove all state descriptions of this handler
         channelStateByChannelUID.forEach((uid, state) -> stateDescProvider.remove(uid));
         super.dispose();
-        // there is a design flaw, we can't clean up our stuff because it is needed by the super-class on disposal for unsubscribing
+        // there is a design flaw, we can't clean up our stuff because it is needed by the super-class on disposal for
+        // unsubscribing
         channelStateByChannelUID.clear();
     }
 
     @Override
     public CompletableFuture<Void> unsubscribeAll() {
-        return CompletableFuture.allOf(channelStateByChannelUID.values().stream().map(ChannelState::stop)
-                .toArray(CompletableFuture[]::new));
+        return CompletableFuture.allOf(
+                channelStateByChannelUID.values().stream().map(ChannelState::stop).toArray(CompletableFuture[]::new));
```

Comment:
This is a whitespace only change. Is the ESH formatter in use?

Reply:
Yes, I tried with all ESH formatter, openHAB formatter (https://www.openhab.org/docs/developer/guidelines.html#java-code) and `mvn spotless:apply`.
They all give me this result.

I now ran `mvn spotless:apply` on the project. There are some more WS-only changes...